src/ui/molecules/checkbox/index.tsx

```diff
@@ -56,17 +43,98 @@ export const Checkbox = styled(CheckboxBase)`
       (1px * var(--woly-main-level) * var(--woly-component-level))
   );
 
-  position: relative;
+  --woly-checkbox-width: 24px;
+  --woly-checkbox-height: 24px;
 
-  display: flex;
-  align-items: center;
-  padding: var(--woly-vertical, 16px) var(--woly-horizontal, 6.4px);
+  padding: var(--woly-vertical, 6.4px) var(--woly-horizontal, 6.4px);
 
-  input {
-    margin: 0;
+  user-select: none;
+
+  [data-block='container'] {
+    display: flex;
+    align-items: center;
+
+    [data-block='checkmark-checked'],
+    [data-block='checkmark-unchecked'] {
```

Comment:
И здесь станет приятнее:
```suggestion
    [data-checkmark] {
```